[2024-04-14T07:06:01.063+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-14T07:06:01.075+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.create_Finance_dataset manual__2024-04-14T05:09:56.705108+00:00 [queued]>
[2024-04-14T07:06:01.081+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.create_Finance_dataset manual__2024-04-14T05:09:56.705108+00:00 [queued]>
[2024-04-14T07:06:01.082+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-14T07:06:01.095+0000] {taskinstance.py:2327} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_Finance_dataset> on 2024-04-14 05:09:56.705108+00:00
[2024-04-14T07:06:01.109+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl', 'create_Finance_dataset', 'manual__2024-04-14T05:09:56.705108+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmp2z861ppp']
[2024-04-14T07:06:01.111+0000] {standard_task_runner.py:91} INFO - Job 57: Subtask create_Finance_dataset
[2024-04-14T07:06:01.122+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=81430) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-14T07:06:01.123+0000] {standard_task_runner.py:63} INFO - Started process 81434 to run task
[2024-04-14T07:06:01.164+0000] {task_command.py:426} INFO - Running <TaskInstance: etl.create_Finance_dataset manual__2024-04-14T05:09:56.705108+00:00 [running]> on host 2ce858e27684
[2024-04-14T07:06:01.397+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='create_Finance_dataset' AIRFLOW_CTX_EXECUTION_DATE='2024-04-14T05:09:56.705108+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-14T05:09:56.705108+00:00'
[2024-04-14T07:06:01.399+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-14T07:06:01.436+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-04-14T07:06:01.454+0000] {base.py:84} INFO - Using connection ID 'my_gcp_conn' for task execution.
[2024-04-14T07:06:01.560+0000] {bigquery.py:470} INFO - datasetId was not specified in `dataset_reference`. Will use default value Finance.
[2024-04-14T07:06:01.562+0000] {bigquery.py:470} INFO - projectId was not specified in `dataset_reference`. Will use default value dataengineer-415510.
[2024-04-14T07:06:01.562+0000] {bigquery.py:481} INFO - Creating dataset: Finance in project: dataengineer-415510 
[2024-04-14T07:06:02.404+0000] {bigquery.py:485} INFO - Dataset created successfully.
[2024-04-14T07:06:02.419+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-14T07:06:02.426+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=etl, task_id=create_Finance_dataset, execution_date=20240414T050956, start_date=20240414T070601, end_date=20240414T070602
[2024-04-14T07:06:02.449+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-14T07:06:02.465+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-14T07:06:02.471+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
